dotCMS/src/main/java/com/dotmarketing/filters/RESTAPIExceptionFilter.java

```diff
@@ -0,0 +1,61 @@
+package com.dotmarketing.filters;
+
+import com.dotcms.repackage.org.apache.commons.httpclient.HttpStatus;
+import com.dotmarketing.exception.DotSecurityException;
+import java.io.IOException;
+import javax.servlet.Filter;
+import javax.servlet.FilterChain;
+import javax.servlet.FilterConfig;
+import javax.servlet.ServletException;
+import javax.servlet.ServletRequest;
+import javax.servlet.ServletResponse;
+import javax.servlet.http.HttpServletResponse;
+
+/**
+ * @author Andre Curione
+ *
+ * Filter class to catch exceptions and standardize responses throughout REST API Resources
+ */
+public class RESTAPIExceptionFilter implements Filter {
+
+    @Override
+    public void init(FilterConfig filterConfig) throws ServletException {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1186)